Source/Urho3D/Scene/Serializable.h

```diff
@@ -186,8 +186,8 @@ template <typename T, typename U> class EnumAttributeAccessorImpl : public Attri
 template <typename T, typename U> class EnumAttributeAccessorFreeImpl : public AttributeAccessor
 {
 public:
-    using GetFunctionPtr = U(*)(const T*);

```

Comment:
[Thoughtful review needed]
How do you use it?

Reply:
Same as with previous attribute change: https://github.com/rokups/Urho3D/blob/master/Source/Tools/Editor/SceneSettings.cpp#L102-L125

Attribute inspector creates dropdowns for enum attributes and custom based on Variant type for anything else. So this is used for utilizing lambdas as getters/setters when creating enum attribute.